### PR TITLE
refactor: rename recurrnace values

### PR DESCRIPTION
**Pull request by turip**
## Overview

Rename WEEKLY/DAILY/MONTHLY/YEARLY recurrence values to WEEK/DAY/MONTH/YEAR.

### CODE REVIEW

File: api/openapi.yaml [hunk 1]
```diff
@@ -1023,8 +1023,8 @@ paths:
         default:
           $ref: "#/components/responses/UnexpectedProblemResponse"
   # TODO: should this be /value to be compatible with all three entitlement types
-  /api/v1/subjects/{subjectIdOrKey}/entitlements/{entitlementIdOrFeatureKey}/value:
-    get:
+  ? /api/v1/subjects/{subjectIdOrKey}/entitlements/{entitlementIdOrFeatureKey}/value
+  : get:
```
Comment (sagikazarmark) [inline api/openapi.yaml:1027]: ?

**Comment by GAlexIHU** [comment]
apart from that 1 thing looks good to me

File: api/openapi.yaml (hunk 1 above)
Comment (turip) [reply api/openapi.yaml:1027]: My editor messes with me, I will sort this out asap.

Comment (turip) [reply api/openapi.yaml]: Seems like VSCode tries to wrap lines with this syntax if they are more than 80 characters, updated my settings.

**Review by GAlexIHU — APPROVED** [review]

**Pull request merged**